README.md

```diff
@@ -30,27 +72,12 @@ sh scripts/test.sh
 In Python:
 
 ```python
-from latex2sympy import process_sympy
+from latex2sympy.latex2sympy import process_sympy
 
 process_sympy("\\frac{d}{dx} x^{2}")
 # => "diff(x**(2), x)"
 ```
 
-- To modify parser grammar, view the existing structure in `PS.g4`.
-- To modify the action associated with each grammar, look into `latex2sympy.py`.
-
-## Examples
```

Comment:
@builtbywill is there a reason we took out the examples from the readme?

Reply:
@lily514 no not really 🤷